flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -299,15 +299,15 @@
 	/**
 	 * Flag indicating whether to start a thread, which repeatedly logs the memory usage of the JVM.
 	 *
-	 * @deprecated use {@link TaskManagerOptions#DEBUG_MEMORY_USAGE_START_LOG_THREAD} instead
+	 * @deprecated use {@link TaskManagerOptions#DEBUG_MEMORY_LOG} instead
 	 */
 	@Deprecated
-	public static final String TASK_MANAGER_DEBUG_MEMORY_USAGE_START_LOG_THREAD = "taskmanager.debug.memory.startLogThread";
+	public static final String TASK_MANAGER_DEBUG_MEMORY_LOG = "taskmanager.debug.memory.log";
```

Comment:
will change.

Reply:
But confused me is that there is no reference to this field in project.

Reply:
There is no reference because the project works against `TaskManagerOptions#DEBUG_MEMORY_USAGE_START_LOG_THREAD`. We keep this field around since it is in a `@Public` class, and thus part of the stable API.

Reply:
Ah. Makes better sense to me.